data-prepper-plugins/mongodb/src/main/java/org/opensearch/dataprepper/plugins/mongo/client/MongoDBHelper.java

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.plugins.mongo.client;
+
+import com.mongodb.ConnectionString;
+import com.mongodb.MongoClientSettings;
+import com.mongodb.client.MongoClient;
+import com.mongodb.client.MongoClients;
+import org.bson.Document;
+import org.bson.conversions.Bson;
+import org.bson.types.BSONTimestamp;
+import org.bson.types.Binary;
+import org.bson.types.Code;
+import org.bson.types.Decimal128;
+import org.bson.types.ObjectId;
+import org.bson.types.Symbol;
+import org.opensearch.dataprepper.plugins.mongo.configuration.MongoDBSourceConfig;
+import org.opensearch.dataprepper.plugins.truststore.TrustStoreProvider;
+
+import java.io.File;
+import java.util.Objects;
+
+import static com.mongodb.client.model.Filters.and;
+import static com.mongodb.client.model.Filters.gte;
+import static com.mongodb.client.model.Filters.lte;
+
+public class MongoDBHelper {
```

Comment:
I see a few helper methods in here. Perhaps rather than starting one large helper class, we can have multiple based on the function. e.g. `MongoDBConnection.getMongoClient(...)`. I'm not quite sure how to categorize the the public methods which are unused. Perhaps `BsonHelper.buildAndQuery(...)`

Reply:
I agree that we could split this class into *Connection and *Query

Reply:
Done